### PR TITLE
Shrink file sizes and improve memory usage (easy steps)

### PR DESCRIPTION
**Pull request by bpbond**
Changes to `trackingC_setup.Rmd`:

Step | Runs | model_output.csv | /run | trk_output.csv | /run
-- | -- | -- | -- | -- | --
(Initial) | 70 | 36.4 | 0.52 | 120.2 | 1.72
Remove unneeded columns | 70 | 6.8 | 0.1 | 37.1 | 0.53
Skip years (=5) | 70 | 1.4 | 0.02 | 7.4 | 0.11

Changes to `trackingC.Rmd`:

Step | Runs | mem_used()
-- | -- | --
(Initial) | 70 | 478
After file-shrink steps above | 70 | 326
Remove unneeded objects | 70 | 313

### CODE REVIEW

File: trackingC_setup.Rmd [hunk 1]
```diff
@@ -226,7 +226,7 @@ tibble(lognorm = ecs_vals_lognorm, transformed = runlist$ECS) %>%
 
 Run the model and store outputs and tracking data.
 
-```{r functions, cache=TRUE, message=FALSE}
+```{r functions, message=FALSE, cache=TRUE}
```
Comment (leeyap) [inline trackingC_setup.Rmd:229]: Trivial, but is there a reason why you changed this?

**Comment by leeyap** [comment]
Thanks for making these changes! I'll try running a 5000 run csv through and see what happens now.

File: trackingC_setup.Rmd (hunk 1 above)
Comment (bpbond) [reply trackingC_setup.Rmd:229]: Didn't mean to; I clicked on the code chunk gear option, and it automatically rearranged them 🤷

**Comment by bpbond** [comment]
I will try to fix this check failure later.

**Comment by bpbond** [comment]
Back to you @leeyap — I'm not sure what has changed from your #55 PR, but `trackingC.Rmd` is breaking at the ggplot call in line 357, because `atm_summary` has zero rows, because `atmosphere_pool` has zero rows, because `trk_output_final` has zero rows...I didn't trace it further back than that.

Pushing a7ac38d38e5fa8f894e4c896a084d2f0668e9cfb fixing the error in `trackingC_setup.Rmd`.

**Comment by bpbond** [comment]
100% of model runs are failing the CMIP temperature test in line 245.

I did 5000 runs without a problem.

**Comment by leeyap** [comment]
I'm not having either of those issues? Looks like only 55 runs are dropped from the temperature tests in total. And, my ggplot in 357 is running just fine. If all of your runs are failing the test, then `trk_output_final` would indeed be empty, but that shouldn't be the case 🤔

**Comment by bpbond** [comment]
Hmm. Whatever is happening for me is also happening on GitHub Actions, hence failing test.

**Comment by leeyap** [comment]
I'm... not sure. I don't really know how to debug if it's working locally?

**Comment by bpbond** [comment]
Just a guess: it [looks like](https://github.com/JGCRI/trackingC/pull/55/files) that in #55 you updated the CO2 file, but not `CMIP6_annual_tas_global.csv`. Is this correct? I'm wondering whether you have an updated version of that file locally that allows the code to run without an error.

**Comment by leeyap** [comment]
Yes, I updated the CO2 file to include the SSPs it was missing previously, but made no changes to the tas file. I don't see any differences in the file on my computer, or between main and this branch. Hmm.

**Comment by bpbond** [comment]
And your local repo is exactly at [332c2e8](https://github.com/JGCRI/trackingC/pull/57/commits/332c2e858e6a34e4409be9b414b3fce8c3e5c6b2) with no uncommitted changes?

**Comment by leeyap** [comment]
Correct!

**Comment by bpbond** [comment]
If you haven't done this already, could you please: 
* Delete all files in `output_files/`
* Re-run `trackingC_setup.Rmd` after doing `Sys.setenv(CI = "true")`
* Re-run `trackingC.Rmd`

Thank you.

**Comment by leeyap** [comment]
image
Well, this is why the test is failing. Hector temperature suddenly looks like this. Not sure if this could be due to the fact that there are so few runs (only 14 per SSP with CI = "true")?

**Comment by bpbond** [comment]
That is indeed. Wow. What the heck? I don't think there's any way this is due to the only-14 issue, but you can try a slightly higher number. 

>Also - it seems that when I deleted the files in output_files in this branch, without committing anything, all output_files on all branches are gone (help?)

Uh, crap, yes. They're not tracked by Git so when you delete them, they're gone. I'm sorry is I inadvertently gave you bad advice! I can send you my big (n=5000) output files, would that help?

**Comment by leeyap** [comment]
Ahh, okay. No worries, but yes please pass along the 5000 file! I'll run it through and see if it's weird, too.

**Comment by bpbond** [comment]
☝️ that's just bizarre. The Hector world is at +2 °C in 2000?!?

**Comment by bpbond** [comment]
OK, sending your way, hang on.

**Comment by bpbond** [comment]
>Set save_every to one to prevent errors

Bummer.

**Comment by bpbond** [comment]
@leeyap are there other things you'd like done here? Please lmk if anything else I should/can do.

**Comment by leeyap** [comment]
@bpbond Just pushed up my last changes. Okay to merge once Rmarkdown check clears.

**Pull request merged**